modules/machineset-creating-non-guaranteed-instances.adoc

```diff
@@ -14,9 +14,12 @@ You can launch a Spot Instance on AWS by adding SpotMarketOptions to your Machin
 ----
 providerSpec:
   value:
-    spotMarketOptions: {} <1>
+    spotMarketOptions: {}
 ----
-<1> You can optionally set the `maxPrice` for the `spotMarketOptions` field to limit the cost of the Spot Instance, for example `maxPrice: '2.50'`. If the `maxPrice` is set, this value is used as the hourly maximum spot price. If not set, the maximum price defaults to charge up to the On-Demand Instance price.
+
+You can optionally set the `spotMarketOptions.maxPrice` field to limit the cost of the Spot Instance. For example you can set `maxPrice: '2.50'`.
```

Comment:
We usually offset optional steps as:
> Optional: You can set the...

Also, s/For example you/For example, you/